frontend/src/app/modules/imports/services/data.service.ts

```diff
@@ -246,8 +265,22 @@ export class DataService {
     if (chartImg !== '') {
       formData.append('chart', chartImg);
     }
-    return this._http.post(`${this.urlApi}/export_pdf/${importId}`, formData, {
+    return this._http.post(`${this.getUrlApi()}/export_pdf/${importId}`, formData, {
       responseType: 'blob',
     });
   }
+
+  getDestinations(): Observable<Array<Destination>> {
+    return this._http.get<Array<Destination>>(
+      `${this.config.API_ENDPOINT}/import/destinations/`,
+      {}
+    );
+  }
+
+  getDestination(code): Observable<Destination> {
+    return this._http.get<Destination>(
+      `${this.config.API_ENDPOINT}/import/destination/${code}`,
+      {}
+    );
```

Comment:
**Suggestion:** The `getDestination` function is used to get a specific destination from the backend. However, it does not handle the case where the backend returns an error. You should add error handling to this function to ensure that any errors from the backend are properly handled. [best practice]
```suggestion
  getDestination(code): Observable<Destination> {
      return this._http.get<Destination>(
          `${this.config.API_ENDPOINT}/import/destination/${code}`,
          {}
      ).pipe(
          catchError(error => {
              console.error('Error fetching destination:', error);
              return throwError(error);
          })
      );
  }
```